pkgs/development/libraries/hunspell/dictionaries.nix

```diff
@@ -250,6 +250,41 @@ let
       };
     };
 
+  mkDictFromBB =
+    { shortName, shortDescription, dictFileName }:
+    stdenv.mkDerivation rec {
+      name = "hunspell-dict-${shortName}-BB-${version}";
+      version = "20131101";
+      folderName = "ru_RU_UTF-8_${version}";
+
+      src = fetchurl {
+        url = "https://bitbucket.org/Shaman_Alex/russian-dictionary-hunspell/downloads/ru_RU_UTF-8_${version}.zip";
```

Comment:
I think if you hardcode a single-language source url, it is fine but then there is no point in defining a function.